src/olympia/files/tests/test_models.py

```diff
@@ -1306,21 +1306,17 @@ def test_permissions(self):
 
 class TestZip(TestCase, amo.tests.AMOPaths):
 
-    def test_zip(self):
-        # This zip contains just one file chrome/ that we expect
-        # to be unzipped as a directory, not a file.
+    def test_zip_python_bug_4710(self):
+        """This zip contains just one file chrome/ that we expect
+        to be unzipped as a directory, not a file.
+        """
         xpi = self.xpi_path('directory-test')
 
-        # This is to work around: http://bugs.python.org/issue4710
-        # which was fixed in Python 2.6.2. If the required version
-        # of Python for zamboni goes to 2.6.2 or above, this can
-        # be removed.
-        try:
-            dest = tempfile.mkdtemp()
-            zipfile.ZipFile(xpi).extractall(dest)
-            assert os.path.isdir(os.path.join(dest, 'chrome'))
-        finally:
-            rm_local_tmp_dir(dest)
+        # This was to work around: http://bugs.python.org/issue4710
+        # which was fixed in Python 2.6.2.
+        dest = tempfile.mkdtemp(dir=settings.TMP_PATH)
+        zipfile.ZipFile(xpi).extractall(dest)
+        assert os.path.isdir(os.path.join(dest, 'chrome'))
```

Comment:
For some reason, I still like to keep that test. It makes sure it doesn't break in the future again.